### PR TITLE
add github search provider

### PR DESCRIPTION
**Pull request by amanzrx4**

### CODE REVIEW

File: src/types/index.ts [hunk 1]
```diff
@@ -95,3 +96,15 @@ export type GithubLoginParams<T extends ApiType> = {
 	keys: KeyOf<GithubApiProvider[T]>[]
 }
 
+type GithubClaimType = 'issues' | 'commits' | 'repositories'
+
+export type GithubParams = {
```
Comment (swetshaw) [inline src/types/index.ts:101]: Where are we using `GithubParams` ?

Comment (amanzrx4) [reply src/types/index.ts:101]: https://github.com/questbook/tls-receipt-verifier/commit/24d748b29d66caed0c61b874b1a956fe85cb0127
We've updated the github provider @swetshaw

File: src/types/index.ts [hunk 2]
```diff
@@ -1,68 +1,69 @@
...
+	| {
+			provider: 'yc-login'
+			parameters: {}
+	  }
+	| {
+			provider: 'github-claim'
+			parameters: GithubParams
+	  }
+	| {
+			provider: 'github-contributed'
```
Comment (swetshaw) [inline src/types/index.ts:18]: Shouldn't we remove this provider? We discussed it just fetches the repositories of a user.

File: src/types/index.ts (hunk 1 above)
Comment (swetshaw) [reply src/types/index.ts:101]: Got it

**Review by swetshaw — APPROVED** [review]

**Pull request merged**